src/applications/travel-pay/tests/e2e/travel-pay.cypress.spec.js

```diff
@@ -55,4 +78,74 @@ describe(`${appName} -- Status Page`, () => {
       .eq(4)
       .should('include.text', 'August 11, 2022');
   });
+
+  it('filters the claims by status and preserves default sort', () => {
+    cy.openFiltersAndUncheckStatuses();
+    cy.selectVaCheckbox('SAVED_checkbox', true);
+    cy.get('va-button[data-testid="apply_filters"]').click({
+      waitForAnimations: true,
+    });
+
+    cy.get('h2[data-testid="travel-claim-details"]').should('have.length', 5);
+    cy.get('h2[data-testid="travel-claim-details"]')
+      .first()
+      .should('include.text', 'August 16, 2023');
+    cy.get('h2[data-testid="travel-claim-details"]')
+      .eq(4)
+      .should('include.text', 'March 27, 2022');
+  });
+
+  it('filters the claims by multiple statuses and preserves default sort', () => {
+    cy.openFiltersAndUncheckStatuses();
+    cy.selectVaCheckbox('SAVED_checkbox', true);
+    cy.selectVaCheckbox('INCOMPLETE_checkbox', true);
+    cy.get('va-button[data-testid="apply_filters"]').click({
+      waitForAnimations: true,
+    });
+
+    cy.get('h2[data-testid="travel-claim-details"]').should('have.length', 6);
+    cy.get('h2[data-testid="travel-claim-details"]')
+      .first()
+      .should('include.text', 'May 15, 2024');
+    cy.get('h2[data-testid="travel-claim-details"]')
+      .eq(5)
+      .should('include.text', 'March 27, 2022');
+  });
+
+  it('resets the filters when button is pressed', () => {
+    cy.openFiltersAndUncheckStatuses();
+    cy.selectVaCheckbox('SAVED_checkbox', true);
+    cy.get('va-button[data-testid="apply_filters"]').click({
+      waitForAnimations: true,
+    });
+    cy.get('va-button[data-testid="reset_search"]').click({
+      waitForAnimations: true,
+    });
+    cy.get('h2[data-testid="travel-claim-details"]').should('have.length', 10);
+    testStatuses.forEach(statusName => {
+      cy.get(`va-checkbox[name=${statusName}_checkbox]`)
+        .shadow()
+        .find('input[type="checkbox"]')
+        .should('be.checked');
+    });
+    cy.get('h2[data-testid="travel-claim-details"]')
+      .first()
+      .should('include.text', 'May 15, 2024');
+  });
+
+  it('preserves sort order when filters are applied', () => {
+    cy.get('select[name="claimsOrder"]').select('oldest');
+    cy.get('va-button[data-testid="Sort travel claims"]').click();
+    cy.openFiltersAndUncheckStatuses();
+    cy.selectVaCheckbox('SAVED_checkbox', true);
```

Comment:
I think I missed something - what is the `SAVED_checkbox`?

Reply:
I see that this test unchecks all the filters, which filters is it enabling in order to test that sort order is preserved?